### PR TITLE
Headings in options #678

### LINKED ISSUE #678: headings in options
**Description of the documentation**
In the options, the `[PLOTTING]` options all have `[FITTING]` as a header.

image

Also, it's not obvious which section each option is in -- having these laid out on one page may be more helpful to a user.

[PLOTTING] should be changed to [OUTPUT], as most of the options don't relate to plotting.

### PR DESCRIPTION
**Pull request by Shanrahan16**
#### Description of Work

Fixes #678

### CODE REVIEW

File: docs/source/users/options/output_option.rst [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
-##############
-Output Options
-##############
+################
+Plotting Options
+################
 
-The output section contains options to control how results are outputted.
+The plotting section contains options to control how results are outputted and presented.
```
Comment (tyronerees) [inline docs/source/users/options/output_option.rst:7]: I think we should merge `Plotting` into `Output`, not `Output` into `Plotting`; plotting is a form of output, but I'd never to think to look for where to set the output directory under a section called `Plotting`.

File: docs/source/users/options/output_option.rst [hunk 2]
```diff
@@ -20,4 +20,123 @@ Default is ``fitbenchmarking_results``
 .. code-block:: rst
 
     [OUTPUT]
```
Comment (tyronerees) [inline docs/source/users/options/output_option.rst:22]: All the headers should be the same now -- either all `[OUTPUT]` or all `[PLOTTING]`

**Review by tyronerees — CHANGES_REQUESTED** [review]
I think we should also merge the `SECTIONS` in `fitbenchmarking/utils/options.py` too -- breaking the convention of one doc page per section for these two sections is likely to confuse users.

**Comment by Shanrahan16** [comment]
I think I also need to update the ini file templates

File: docs/source/extending/options_extend.rst [hunk 3]
```diff
@@ -13,7 +13,7 @@ The options used can be changed using an `.ini` formatted file
 FitBenchmarking.
 
 To add a new option to one of the five sections ``FITTING``, ``MINIMIZERS``,
```
Comment (tyronerees) [inline docs/source/extending/options_extend.rst:15]: I love how the number of sections is now correct, since you've taken one out 😆

**Review by tyronerees — APPROVED** [review]
Works for me, and looks much cleaner -- thanks!

**Pull request merged**